packages/client/components/EstimatePhaseArea.tsx

```diff
@@ -52,14 +55,28 @@ const containerStyle = {
   height: '100%'
 }
 
-const EstimatePhaseArea = () => {
+interface Props {
+  meeting: EstimatePhaseArea_meeting
+}
+const EstimatePhaseArea = (props: Props) => {
+  const {meeting} = props
+  const {localStage} = meeting
   const [activeIdx, setActiveIdx] = useState(1)
   const isDesktop = useBreakpoint(Breakpoint.SIDEBAR_LEFT)
 
   const onChangeIdx = (idx) => {
     setActiveIdx(idx)
   }
 
+  const getVotedUserEl = (_userId: string) => {
+    // mock Element
+    return {
+      getBoundingClientRect() {
```

Comment:
@ackernaut here's the missing piece where our branches will have to merge. basically, i need this function to return the element of that user's avatar in the voted users array.